### PR TITLE
[cli] Adds boilerplate for --local_image flag

### PR DESCRIPTION
**Pull request by ser-io**

### CODE REVIEW

File: pkg/cli/cvd.go [hunk 1]
```diff
@@ -56,9 +61,12 @@ func newCVDCommand(cfgFlags *configFlags) *cobra.Command {
 	create.Flags().StringVar(&createFlags.Host, hostFlag, "", "Specifies the host")
 	create.MarkFlagRequired(hostFlag)
 	create.Flags().StringVar(&createFlags.BuildID, buildIDFlag, "", "Android build identifier")
-	create.MarkFlagRequired(buildIDFlag)
 	create.Flags().StringVar(&createFlags.Target, targetFlag, "aosp_cf_x86_64_phone-userdebug",
 		"Android build target")
+	create.Flags().BoolVar(&createFlags.LocalImage, localImageFlag, false,
+		"Builds a CVD with an image files locally built, the required files are https://cs.android.com/android/platform/superproject/+/master:device/google/cuttlefish/required_images")
```
Comment (jemoreira) [inline pkg/cli/cvd.go:67]: nit: "with image files built locally"

Comment (jemoreira) [inline pkg/cli/cvd.go:67]: Technically, the files don't need to be "built" locally, they just need to be available in your local host. For example you could download them from some website other than the android build and run remotely.

Comment (jemoreira) [inline pkg/cli/cvd.go:67]: The files listed in that page don't include `cvd-host-packages.tar.gz`.

**Review by jemoreira — APPROVED** [review]

File: pkg/cli/cvd.go (hunk 1 above)
Comment (ser-io) [reply pkg/cli/cvd.go]: done

Comment (ser-io) [reply pkg/cli/cvd.go]: appended cvd-host-packages.tar.gz to the description.

Comment (ser-io) [reply pkg/cli/cvd.go]: yes, we would handle such scenario with a new `local_image_dir` flag.

**Review by jemoreira — APPROVED** [review]

**Pull request merged**